packages/portal/docs/style/BootstrapVue/Card.md

```diff
@@ -0,0 +1,99 @@
+Customised BootstrapVue card component. Currently it is not directly used as in the examples, but could be and forms the base of other card components.
+
+Docs: https://bootstrap-vue.org/docs/components/card
+
+Used by:
+- ContentCard
```

Comment:
Link to this component in the styleguide.

Reply:
Done.